release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1201,6 +1201,132 @@ For more information about the unsupported, community-maintained, version of the
 [id="ocp-release-note-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* When upgrading {gcp-short} clusters that use a boot disk that is not compatible with UEFI, you cannot enable shielded VM support. 
+Previously, this prevented the creation of new machines. 
+With this release, disks with known UEFI incompatiblity have shielded VM support disabled. 
```

Comment:
```suggestion
With this release, disks with known UEFI incompatiblity have shielded-VM support disabled. 
```
I assume we mean "support for a shielded VM", as opposed to VM support that is shielded.

Reply:
It's actualyl a feature name - I'll check but I think it might be "Shielded VM". Good catch!